packages/app/validation/users.js

```diff
@@ -3,15 +3,16 @@ const Joi = require('@hapi/joi')
 const schemas = {
   register: Joi.object({
     name: Joi.string().required(),
+    surname: Joi.string().required(),
     password: Joi.string()
       .min(6)
       .max(30)
       .required(),
-    password2: Joi.ref('password'),
-    role: Joi.string().optional(),
-    company: Joi.string().required(),
-    emailHash: Joi.string().required(),
-    invitationCode: Joi.string().required()
+    // password2: Joi.ref('password'),
+    // role: Joi.string().optional(),
+    // company: Joi.string().required(),
+    email: Joi.string().required(),
+    // invitationCode: Joi.string().required()
```

Comment:
Do we really want to remove this validation? Or this is just not part of the submission anymore?

Reply:
In fact, this was part of the old implementation and we don’t have these fields anymore (invitation code, role, and company) but I needed you to give me the ok before I delete them for real.

Reply:
Got you yeah looks good to get rid of them